src/mbgl/map/transform.cpp

```diff
@@ -313,6 +313,107 @@ void Transform::_easeTo(const CameraOptions& options, double new_scale, double n
     }
 }
 
+void Transform::flyTo(const CameraOptions &options) {
```

Comment:
/cc @mourner @ansis here maybe for some backup. I've rev-eng'd some of the projection stuff over time, too, so I'll take a look when I review in depth.